src/sparseml/exporters/transforms/onnx_transform.py

```diff
@@ -61,5 +82,11 @@ def post_validate(self, model: ModelProto) -> ModelProto:
         :param model: The input ONNX model to be validated
         :return The validated ONNX model
         """
+        model.graph.node.extend(self._nodes_to_add)
+        for node in self._nodes_to_delete:
+            model.graph.node.remove(node)
```

Comment:
see: https://github.com/neuralmagic/sparseml/blob/6cf2b086441c02647915357ef6ba46b2395de8b2/src/sparseml/onnx/utils/graph_editor.py#L193

Reply:
I originally used this function, but it was doing some weird things in the tests like removing nodes it shouldn't

Reply:
Oh wow, weird